app/controllers/api/v2/searches_controller.rb

```diff
@@ -79,17 +79,21 @@ def affiliate_docs_search_class
       end
 
       def handle_query_routing
-        return unless search_params[:query].present? and query_routing_is_enabled?
+        # Update the docs with any changes
+        # app/views/sites/api_instructions/_show_web_md.html.haml
```

Comment:
I don't think we need to call out our internal documentation, but we do need to call out the requirement to update https://open.gsa.gov/api/searchgov-results/. However, let's put that admonition at the top of app/views/sites/api_instructions/_show_web_md.html.haml, rather than in the routed query code.

Reply:
Right, that was what I was supposed to do. Embarrassed I misremembered at the end of the day.